app/notifier.py

```diff
@@ -7,8 +7,10 @@
 while True:
     for playerid in playerids:
         if is_online(playerid):
-                print(f"Player {playerid} is online")
-                send_sms(playerid)
-                playerids.remove(playerid)
+            print(f"Player {playerid} is online")
+            send_sms(playerid)
+            playerids.remove(playerid)
+        elif is_online(playerid)!=True:
+            playerids.append(playerid)
```

Comment:
this checks if a player is offline and then adds them to the array

Let's try with a real person

Ava is offline
 if is_online(Ava):
  # pass because I am offline
elif is_online(Ava) !=True
   # I am now here because I'm offline
   playerids.append(Ava) #now I'm added again to the array so the array now has my ID twice